[2023-08-04T19:32:42.499+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: basic_etl_dag.transform_task manual__2023-08-04T19:32:32.958313+00:00 [queued]>
[2023-08-04T19:32:42.506+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: basic_etl_dag.transform_task manual__2023-08-04T19:32:32.958313+00:00 [queued]>
[2023-08-04T19:32:42.506+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-08-04T19:32:42.506+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-08-04T19:32:42.506+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-08-04T19:32:42.550+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): transform_task> on 2023-08-04 19:32:32.958313+00:00
[2023-08-04T19:32:42.554+0000] {standard_task_runner.py:55} INFO - Started process 61824 to run task
[2023-08-04T19:32:42.559+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'basic_etl_dag', 'transform_task', 'manual__2023-08-04T19:32:32.958313+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/basic_etl_dag.py', '--cfg-path', '/tmp/tmppc6lt3y0']
[2023-08-04T19:32:42.561+0000] {standard_task_runner.py:83} INFO - Job 3: Subtask transform_task
[2023-08-04T19:32:42.634+0000] {task_command.py:388} INFO - Running <TaskInstance: basic_etl_dag.transform_task manual__2023-08-04T19:32:32.958313+00:00 [running]> on host codespaces-657b09
[2023-08-04T19:32:42.848+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=basic_etl_dag
AIRFLOW_CTX_TASK_ID=transform_task
AIRFLOW_CTX_EXECUTION_DATE=2023-08-04T19:32:32.958313+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-04T19:32:32.958313+00:00
[2023-08-04T19:32:42.920+0000] {logging_mixin.py:137} WARNING - /workspaces/hands-on-introduction-data-engineering-4395021/airflow/dags/basic_etl_dag.py:24 SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2023-08-04T19:32:42.934+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-08-04T19:32:42.939+0000] {taskinstance.py:1318} INFO - Marking task as SUCCESS. dag_id=basic_etl_dag, task_id=transform_task, execution_date=20230804T193232, start_date=20230804T193242, end_date=20230804T193242
[2023-08-04T19:32:43.011+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-08-04T19:32:43.027+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
